### ISSUE TITLE
Change title to highlight the FAIR aspect

### ISSUE THREAD

**Issue by pcmasuzzo** [OpenScienceMOOC/Module-4-Open-Research-Data#2]
Given we are about to start content develop for this module, I went through the scope and goals and I came to the conclusion that perhaps the title of the module is a bit misleading. 

_**Open Data is not always possible and Open and FAIR do not necessarily overlap.**_

If you do not open up the data (for many reasons), your data can and should still be as FAIR as possible.
I am saying this because a lot of researchers in disciplines like Medicine will perhaps think: "I cannot do open data, there's nothing for me in this module"

Bottom line: I would call the module

- FAIR research data or
- FAIR and OPEN research data.

**Comment by ivogrigorov** [comment]
I’d totally second the suggestion! The module can reflect the opportunity to capitalize on either or Fair/OD. This should potentially DOUBLE the audience: those than need FAIR but have no interest in OD, and Fair-OD keen. Possibly an ambitious title might go in the direction of “Synergies Between FAIR & Open Data”

**Comment by Protohedgehog** [comment]
I guess this is up to @jcolomb  now, but I would rather not, as it means the quite laborious task of changing all the logos and wherever this module has been mentioned around the Web now!

**Comment by jcolomb** [comment]
As for a scientific paper, we will probably talk about changing the title once the module is finished. :)
my 2 cent now, I like short titles and we are in the "open science mooc", so "open research data" seems a straightforward titlte.

So we keep it like this for now, and will discuss it once the content is set.

**Comment by pcmasuzzo** [comment]
"FAIR" enough :P
